codalab/rest/bundles.py

```diff
@@ -479,6 +541,42 @@ def _fetch_bundle_location(bundle_uuid: str, bundle_store_uuid: str):
     return BundleLocationListSchema(many=True).dump(bundle_location).data
 
 
+@post(
+    '/bundles/<bundle_uuid:re:%s>/locations/blob' % spec_util.UUID_STR,
+    apply=AuthenticatedProtectedPlugin(),
+)
+def _update_bundle_location(bundle_uuid: str):
+    """
+    Finalize after uploading a file to the blob storage.
+
+    Query parameters:
```

Comment:
Because this is a POST request, I think these should be part of the JSON body, rather than query string parameters.